examples/bpmn-utlis.js

```diff
@@ -0,0 +1,577 @@
+// Inspired from 'Procurement Processes with Error Handling - Stencil Trisotech 3 pages.bpmn' of https://www.omg.org/cgi-bin/doc?dtc/10-06-02.zip
```

Comment:
:question: can we rename this file into `bpmn-diagrams.js` as it only provident BPMN diagram content?

Reply:
Done